prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="file:///Users/nicholasprodehl/repos/prep-91621-code-solutions/prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
To keep our links independent of the computer they were authored on, we want to use a relative path to the index file.

`href="../index.html"`